src/ray/raylet/main.cc

```diff
@@ -818,7 +809,17 @@ int main(int argc, char *argv[]) {
                                                    is_head_node,
                                                    *node_manager);
 
-    // Initialize event framework.
+    // Initialize stats. This should be done after the node manager is initialized.
```

Comment:
```suggestion
    // Initializing stats should be done after the node manager is initialized because <explain why>.
    // Metrics exported before this call will be buffered until `Init` is called.
```